kokkoro/util.py

```diff
@@ -19,6 +20,9 @@
 from kokkoro import config
 from kokkoro.typing import Iterable
 
+for font_file in kokkoro.config.FONT_PATH.values():
+    font_manager.fontManager.addfont(os.path.expanduser(font_file))
+
```

Comment:
I think this is a part of configuration. Move it to `config_example/__init__.py` is better.

Reply:
Agreed!